root/static/scripts/edit/URLCleanup.js

```diff
@@ -1126,6 +1126,15 @@ const CLEANUPS = {
           case LINK_TYPES.discogs.release_group:
             return {result: prefix === 'master'};
           case LINK_TYPES.discogs.release:
+            if (prefix === 'master') {
+              return {
+                error: l(
+                  `Discogs master links group several releases,
+                   and should be added to release groups instead.`,
```

Comment:
```suggestion
                   thus should be added to release groups instead.`,
```

Reply:
"thus" sounded too fancy to me, but changed to "so this should be added to the release group instead" :)